sys-devel/llvm-common/files/50llvm-mir-mode-gentoo.el

```diff
@@ -0,0 +1,4 @@
+;;; llvm-mir-mode site-lisp configuration
```

Comment:
Do we actually have to keep 3 files in FILESDIR? That's a horrible waste of inodes.

Reply:
Probably not, will see what can be done instead

Reply:
Emacs packages tend to do this in FILESDIR but I don't know why they're not inlined as they're almost always this small.

Reply:
done

Reply:
@thesamesam, @mgorny: do you think something like this should be done for other Emacs packages too? The only negative I see here is if you have a bunch of ebuild versions and need to change the sitefile you can't just change the file in filesdir.

Coudldn't use elisp-site-file-install <<- EOF ... EOF here since it uses the filename when installing.

Reply:
If you don't mind me asking, are these files generally uniform? Perhaps even a better idea would be to add a simple function to the eclass that would take name, description and create the file itself.

Reply:
They almost always look like this - two lines for search path & optionally autoload.